README.md

```diff
@@ -1,6 +1,6 @@
 # Helium Miner Software
 
-This repository includes the main [docker-compose.yml](https://github.com/NebraLtd/helium-miner-software/blob/master/docker-compose.yml) file that powers the Nebra miners.
+This repository includes the main docker-compose.yml(Follow the steps to generate this [here](#generating-docker-compose-file)) file that powers the Nebra miners.
```

Comment:
Missing space before bracket

Also could change includes to generates to further clarify

Reply:
Updated the text here, sounds a lot better with generates instead of includes